gpu-service/entrypoint.sh

```
@@ -0,0 +1,58 @@
+    from qdrant_client import QdrantClient
+    print('✓ Qdrant client imported successfully')
+except Exception as e:
+    print('✗ Qdrant client import failed:', e)
+"
+
+# Set environment variables for better stability
+export TF_CPP_MIN_LOG_LEVEL=2
+export CUDA_VISIBLE_DEVICES=${CUDA_VISIBLE_DEVICES:-0}
+
+echo "=== Starting Application ==="
+echo "Command: $@"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)